### DISCUSSION TITLE
招聘FAMLAND社区干事启事

### DISCUSSION THREAD

**Discussion by yanzhiwuli** [famland/website#160] [General]
famland社区工作需要，现招募社区专职勤杂工一名
主要负责以下工作：
1. 负责社区电报群维护
2. 负责Famland日常数据统计
3. 组织各种会议，议题
4. 社区需求统计
5. 负责社区微信群管理，信息收集与发布，反馈问题进度并跟踪汇报
6. 各种议案收集，发起
7. 项目推广等。
8. 组织合约升级后正式部署前的测试。
待遇：：工资暂定600U每月。
要求：有责任心，对FAMLAND项目非常熟悉和热爱。
报名截止时间2022年11月11日12时

**Comment by otto-001** [comment]
kongpingz.m 报名
参与Famland一年以来，作为原始田投资人，也一直以自愿者身份为社区提供服务，制作了一些宣传图和视频，制作了简单的电报机器人和一些社区运营的工具，参与了每次合约部署前的测试。

**Reply by babaoshu9098** [reply]
地址baboshu1314时间充裕，会电脑，两个农场主，熟悉Famland的所有玩法，八宝树资深老玩家，应聘社区干事
